app/controllers/employees/parameter_sanitizer.rb

```diff
@@ -0,0 +1,9 @@
+module Employees
+  class ParameterSanitizer < Devise::ParameterSanitizer
```

Comment:
It's a little confusing to have non-controller class in `controllers` dir.

I think you could have a simpler solution with:
```
before_action :configure_permitted_parameters, if: :devise_controller?


def configure_permitted_parameters
    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:first_name, :last_name, :email, :password) }
    devise_parameter_sanitizer.permit(:account_update) do |u|
        u.permit(:first_name, :last_name, :email, :password, :current_password)
    end
end


```

Reply:
I have changed it 💯